<launch> 
	<node name="soundplay_node" pkg="sound_play" type="soundplay_node.py"/>

	<node pkg="elektron_navigation" type="goal_status.py" name="goal_status" output="screen">
		<param name="snd_failed" type="string" value="$(find elektron_navigation)/sounds/fail.wav"/>
		<param name="snd_accepted" type="string" value="$(find elektron_navigation)/sounds/accepted.wav"/>
		<param name="snd_success" type="string" value="$(find elektron_navigation)/sounds/success.wav"/>
	</node>
</launch>